app/views/users/show.html.erb

```diff
@@ -1,28 +1,29 @@
 <div class="user-profile__user-info">
   <%= @user.name %>
-  <% if @current_user == @user %>
-    <ul class="user-profile__controls">
-      <li class="user-profile__controls--link">
-        <%= link_to "Edit User", edit_user_path(@user.id) %>
-      </li>
-      <li class="user-profile__controls--link">
-        <%= link_to "Delete User", user_path(@user.id), method: :delete, data: { confirm: "Are you sure?" } %>
-      </li>
-    </ul>
-  <% end %>
+  <ul class="user-profile__controls">
+    <%= @user_profile_presenter.display_profile_options %>
```

Comment:
think about reading the method name while looking at this file, is that enough to tell you what it's going to put into the view?